src/Pos/Types/Block/Functions.hs

```diff
@@ -183,16 +186,20 @@ recreateMainBlock _gbHeader _gbBody _gbExtra = do
         Left err -> fail $ toString err
     pure gb
 
+-- | Dummy implementation of make genesis extra header data.
+mkGenesisEHD :: () -> GenesisExtraHeaderData
+mkGenesisEHD = GenesisExtraHeaderData . mkAttributes
+
 -- | Smart constructor for 'GenesisBlockHeader'. Uses 'mkGenericHeader'.
 mkGenesisHeader
     :: BiSsc ssc
     => Maybe (BlockHeader ssc)
     -> EpochIndex
     -> Body (GenesisBlockchain ssc)
+    -> ExtraHeaderData (GenesisBlockchain ssc)
```

Comment:
Why is it needed if it's always `GenesisExtraHeaderData $ mkAttributes ()`? Am I missing something?